cli/commands/benchmark-url.mjs

```diff
@@ -245,6 +247,10 @@ function outputResults( opt, results ) {
 			tableRow.push( metricAvgMs );
 		}
 
+		for ( let i = tableRow.length; i < headings.length; i++ ) {
+			tableRow.push( '' );
+		}
```

Comment:
Hmm, this doesn't look right. In what situation would there be missing columns? _If_ there are, we would need to match them with the right index based on the heading, not just fill up empty columns at the end, since otherwise the metric values would be associated with the wrong metric heading.

Reply:
@felixarntz in my analysis I have a mix of URLs, part of them lead to empty PHP scripts, another part to WordPress pages. Responses for simple PHP scripts don't contain Server-Timing headers, thus I need to add empty strings to make sure I have equal number or rows for all URLs.

Reply:
You are right. Updated.